### ISSUE TITLE
Set up automated builds to create binary releases via GitHub Actions

### ISSUE THREAD

**Issue by Duckwhale** [RagnarokResearchLab/RagLite#14]
Needs some support from the runtime (just adding the bundling code, more or less). Not a high priority right now, though.

**Comment by Duckwhale** [comment]
Isn't this just `evo build` (in a separate folder), zip up the root dir and publish the release? More advanced bundling isn't needed.

Well, I guess that will pack up the WebView prototype... so a little bit of re-arranging might be needed as well.

**Comment by Duckwhale** [comment]
Yep... quick prototype done, works as expected but isn't very useful. Will wait with completing this until more features are in.

See https://github.com/RagnarokResearchLab/RagLite/tree/14-basic-release-builds

**Comment by rdw-software** [comment]
Addendum: There's additional resources that would have to be extracted from the VFS, such as shader sources and font faces.

While it's possible to do this manually, it's a bit tedious. Improved `vfs` APIs are being added, so defer this until `v0.20.0` is ready.
